pkg/p2p/libp2p/connections_test.go

```diff
@@ -543,7 +543,9 @@ func TestConnectRepeatHandshake(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if _, err := s2.HandshakeService().Handshake(ctx, libp2p.NewStream(stream), info.Addrs[0], info.ID); err != nil {
+	s := s2.WrapStream(stream)
```

Comment:
Since `s` is only used in the `Handshake` method, I would suggest nesting it there.